backend/de.metas.business/src/main/java/de/metas/product/impl/ProductDAO.java

```diff
@@ -582,4 +582,15 @@ public int getGuaranteeMonthsInDays(@NonNull final ProductId productId)
 		final I_M_Product product = getById(productId);
 		return GroupTemplateId.optionalOfRepoId(product.getC_CompensationGroup_Schema_ID());
 	}
+
+	@Override
+	public void clearIndividualMasterDataFromProduct(final ProductId productId)
+	{
+		queryBL.createQueryBuilder(I_M_Product.class)
+				.addEqualsFilter(I_M_Product.COLUMNNAME_M_Product_ID, productId)
+				.create()
+				.updateDirectly()
+				.addSetColumnValue(I_M_Product.COLUMNNAME_M_AttributeSetInstance_ID, 0)
```

Comment:
instead of ZERO pls use:
* -1 if u want to nullify the value
* or AttributeSetInstanceId.NONE.getRepoId() if u want to set it to NONE

Reply:
also, is there any reason for using direct update?